README.md

```diff
@@ -76,16 +75,18 @@ const iotagentLib = require('iotagent-node-lib');
 ```
 
 Information about how to configure the Library can be found at the corresponding section of the
-[Installation & Administration Guide](doc/installationguide.md).
+[Installation & Administration Guide](doc/admin.md).
 
 ## Usage
 
-This library has no packaging or build processes. The [Getting Started](doc/getting-started.md) is a good place to
-start. Usage of the library is explained in the [User & Programmers Manual](doc/usermanual.md).
+This library has no packaging or build processes. The [Getting Started](./doc/getting-started.md) is a good place to
+start. You can also review the [API documentation](./doc/api.md) for a full list of the available functions.
+
+If you plan to use the library in your own IoT Agent, you should read the [Developer Guide](./doc/devel/development.md).
+You can also review the [Architecture](./doc/devel/architecture.md) documentation and
+[Northbound API](./doc/devel/northbound-api.md).
 
--   Details of the architecture of an IoT Agent be found [here](doc/architecture.md).
--   Further Advanced topics can be found [here](doc/advanced-topics.md).
--   The following features are listed as [deprecated](doc/deprecated.md).
+The following features are listed as [deprecated](doc/deprecated.md).
```

Comment:
Comparing old and new links... this means that old advanced-topics.md is the new nortbound-api.md?

Reply:
NTC